content/mqtt/index.textile

```diff
@@ -0,0 +1,113 @@
+
+On receiving this, the client is recommended to get a new token, then disconnect and reconnect with the new token themselves. If this is not done, once the token expires, the server will abruptly disconnect the connection.
+
+An example of this with the NodeJS "mqtt package":https://www.npmjs.com/package/mqtt would be:
+
+bc[nodejs]. {
+  var client = mqtt.connect('mqtts:mqtt.ably.io', options);
+  client.subscribe("[mqtt]tokenevents", {
+    /* Create a new token called 'NEW_TOKEN' */
+
+    client.end();
+    options.username = NEW_TOKEN;
+    client = mqtt.connect('mqtts:mqtt.ably.io', options);
+  });
+}
+
+h3(#ssl). SSL
+
+We support both SSL and non-SSL connections (which use a different port, see above), but strongly recommend using SSL wherever possible. If you are not using SSL, note that the same restrictions apply as if you were using an Ably client without SSL. That is, "connection attempts using basic authentication (i.e. an API key) are disallowed":https://support.ably.io/solution/articles/3000065731-error-code-40103-invalid-use-of-basic-auth-over-non-tls-transport, and any "namespaces which you've enabled 'require TLS' on":https://support.ably.io/solution/articles/3000030058-what-is-a-channel-namespace-and-how-can-i-use-them will be inaccessible. If this is a problem for you, please do let us know at support@ably.io and we'll see what we can do.
+
+h2(#adapter-info). Adapter information
+
+* A protocol adapter inevitably adds some latency. Using the adapter will be a little slower than using Ably native client libraries. Typically the impact is in the low milliseconds
+* Behind the scenes, the adapter just uses the normal Ably service, so there is no problem with using MQTT and Ably client libraries side by side. You can mix and match as you like; for example, using MQTT on your IoT devices, but using the Ably Realtime API on your servers
+* While the adapter can be useful for devices which need to use MQTT, there are many benefits to using the Ably client libraries (e.g. "continuity guarantees":https://support.ably.io/solution/articles/3000044639-connection-state-recovery, "fallback host support":https://support.ably.io/solution/articles/3000044636-routing-around-network-and-dns-issues, "history":https://www.ably.io/documentation/realtime/history, "presence":https://www.ably.io/documentation/realtime/presence) require the use of the Ably client libraries. As a result, if a native Ably library is available for your platform, we recommend you consider using the Ably client libraries instead.
+
+h2(#tutorials). Tutorials
+
+If you would like to see an example of using the MQTT tutorial, we recommend you take a look at our "MQTT snake tutorial":https://www.ably.io/tutorials/mqtt-snake.
```

Comment:
Perhaps an unnoticed typo. Maybe write "If you would like to see the MQTT protocol adapter in action, we recommend.."